crates/rpc/rpc-types/src/eth/block.rs

```diff
@@ -131,6 +131,9 @@ pub struct Header {
     /// Excess blob gas
     #[serde(rename = "excessBlobGas", skip_serializing_if = "Option::is_none")]
     pub excess_blob_gas: Option<U64>,
+    /// Parent beacon block root
+    #[serde(rename = "parentBeaconBlockRoot", skip_serializing_if = "Option::is_none")]
```

Comment:
are tehse renames needed? doesn't the serde name camel case cut it?

Reply:
they aren't needed, missed the camel case rename. the only one that needs the rename is the uncles hash